static/js/reactions.js

```diff
@@ -62,6 +73,15 @@ exports.toggle_reaction = function (message_id, emoji_name) {
     popovers.hide_reactions_popover();
 };
 
+var reaction_show_list = []; // local reaction_show_list
+
+exports.render_reaction_show_list = function () {
+    var reaction_list = $(".reaction-popover-reaction");
+    reaction_show_list = reaction_list.filter(function () {
+        return this.style.display === "block" || this.style.display === "";
+    }).toArray();
+};
```

Comment:
You could probably pretty easily extract the "maintain reaction_show_list" part into a separate commit that's the first half of "reactions.js: Create reaction_navigate function." and make this a lot more readable.